trunk/NDHMS/Land_models/NoahMP/phys/module_sf_noahmplsm.F

```diff
@@ -9268,12 +9279,18 @@ subroutine read_mp_veg_parameters(DATASET_IDENTIFIER)
     ISICE_TABLE        = -99999
     ISCROP_TABLE       = -99999
     EBLFOREST_TABLE    = -99999
+    NATURAL_TABLE      = -99999
+    LOW_DENSITY_RESIDENTIAL_TABLE   = -99999
+    HIGH_DENSITY_RESIDENTIAL_TABLE  = -99999
+    HIGH_INTENSITY_INDUSTRIAL_TABLE = -99999
+
+    inquire( file='MPTABLE.TBL', exist=file_named ) 
```

Comment:
Acceptable on WCOSS?
Works on WCOSS?
How to test on WCOSS?

Reply:
@donaldwj 
I think we should just accept this and deal with the issue later, when the time comes.

Reply:
If NCO has an issue with this they will say something when we try to hand off the code, I haven't had a chance to determine if this violates any of their rules.

Reply:
I will take that as a "go ahead and merge  and we will deal with issues on WCOSS later".

Reply:
Would this ever return true on WCOSS? If it's not opened by name, it shouldn't, right? I think the way it's handled on WCOSS the actual on-disk filename will be something akin to "`fort.15`" and therefore it should be ok. I'm alright with the shoot-first, ask-questions-later approach with WCOSS for now.